website/docs/cloud/cli.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 2
+---
+
+# Run via CLI
+
+:::info
+Flashlight Cloud supports only Maestro for now, if you wish to have support for more testing frameworks, reach out on Slack or via mail
+:::
+
+## Usage
+
+Create your API key [here](https://app.flashlight.dev/api-key) and set it as an environment variable:
+
+```bash
+export FLASHLIGHT_API_KEY="fl_xxxxx"
+```
+
+Create your Maestro test file, for instance `start.yml`:
+
+```yml
+appId: com.example
+---
+- launchApp
+- assertVisible: .*KILL JS.*
+```
+
+Then run:
+
+```bash
+flashlight cloud --app example.apk --test start.yml --duration 10000
+```
+
+:::tip 
+We'll soon support .aab files via the CLI
+:::
```

Comment:
Maybe show supported extensions. Since the arg name is "app" it's not clear that we only support apks and not .zip, .ipa, .app, etc